edkahara/iReporter
# Ft create v2 create report endpoint #162423040

__What does this PR do__
Create the create a report endpoint for v2 of iReporter

__Description of tasks to be completed.__
Create an api endpoint that facilitates the creation of a new report. Test the api endpoint.

__How should it be manually tested?__
Clone the project. Check out the ft-create-v2-create-report-endpoint-#162423040 branch. Start the environment using `env\Scripts\activate` and `env.bat` to set up environment variables. Run flask run on the terminal and go to Postman. To perform the tests run nosetests.

__Relevant pivotal tracker stories.__
#162423040
